test/src/components/serviceInsights/serviceInsights.spec.jsx

```diff
@@ -116,5 +117,28 @@ describe('<ServiceInsights/>', () => {
 
             sinon.assert.calledWith(timeWindow.toTimeRange, '888');
         });
+
+        it('shoudl render an error message when no traces found', () => {
```

Comment:
shoudl -> should

Reply:
Fixed here: https://github.com/nathanwalther/haystack-ui/pull/3